app/furniture/marketplace/cart_products/_cart_product.html.erb

```diff
@@ -4,7 +4,7 @@
   <td class="w-full max-w-0 py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:w-auto sm:max-w-none sm:pl-6">
     <%- if product.photo.present? %>
       <figure>
-        <%= image_tag product.photo.variant(resize_to_limit: [550, 550]).processed.url, class: "rounded-lg"
+        <%= image_tag product.photo.variant(resize_to_limit: [150, 150]).processed.url, class: "rounded-lg"
```

Comment:
We could add named variants if we'd like: https://www.bigbinary.com/blog/rails-7-adds-ability-to-use-predefined-variants#rails-7-onwards

But I also think that keeping the sizes in the UI layer (`View` or `ViewComponent` are both good)

Reply:
We decided to leave it in the views and components for now, keep it in the views instead of the model.